p2p/mdns.go

```diff
@@ -22,89 +22,88 @@ import (
 
 	log "github.com/ChainSafe/log15"
 	"github.com/libp2p/go-libp2p-core/peer"
-	libp2pdiscovery "github.com/libp2p/go-libp2p/p2p/discovery"
+	discovery "github.com/libp2p/go-libp2p/p2p/discovery"
 )
 
-const mdnsPeriod = time.Minute
+const MdnsPeriod = time.Minute
```

Comment:
not sure if this needs to be exported, not a huge deal tho

Reply:
Yea, I was not sure either.